views/sign_in.erb

```diff
@@ -1,15 +1,20 @@
-<h1>Sign in to your Makers BnB account</h1>
+<h2>Sign in to your Makers BnB account</h2>
 
 <%= flash[:incorrect_details] %>
 
 <form action="/sign-in-input" method="post">
-  <label for="email"> email
+  <label for="email"> Enter your email</label>
+  <br>
   <input type="text" name="email">
-    <label for="password"> password
+  <br>
+  <label for="password"> Enter your password</label>
+  <br>
   <input type="password" name="password">
+  <br>
   <input type="submit" value="Log in">
 </form>
-
+<br>
 <form action="/sign-up" method="get">
+  Don't have an account?
```

Comment:
Can you put this in a `<p></p>` tag please?

Reply:
Good spot! Will do that now.